### ISSUE TITLE
Addv2 op in BN caused QuantizationSimModel failed

### ISSUE THREAD

**Issue by cookcodes** [quic/aimet#432]
My model is trained using Keras. and then export to chkpt, then import it tensorflow 1.x

But when I tried to quantize the model, it failed at fusedbatchnorm.py",  in get_beta_read_var_op_tensor
assert bn_op.type in ['FusedBatchNormV3', 'Mul']

I added some logs in sub_graph_match.py, here are the informaiton:

2021-02-02 10:06:03,003 - ConnectedGraph - DEBUG - matched op:AddV2, batch_normalization_1/batchnorm_1/add_1
2021-02-02 10:06:03,003 - ConnectedGraph - DEBUG - template_pattern_type:BN_non_fused_keras_with_training_False
2021-02-02 10:06:03,003 - ConnectedGraph - DEBUG - op_type:BatchNorm
2021-02-02 10:06:03,003 - ConnectedGraph - DEBUG - module_regex:['(.+)/batchnorm/mul_1$']
2021-02-02 10:06:03,003 - ConnectedGraph - WARNING - Unable to identify module name, using name of first op as module name: batch_normalization_1/batchnorm_1/add_1
2021-02-02 10:06:03,003 - ConnectedGraph - WARNING - Unable to identify associated op of module, setting first op in ops_list as associated op.

And then:
2021-02-02 10:06:03,169 - ConnectedGraph - DEBUG - handle op:BatchNorm, batch_normalization_3/batchnorm_1/add_1
2021-02-02 10:06:03,169 - ConnectedGraph - DEBUG - tf_op:AddV2, batch_normalization_3/batchnorm_1/add_1

Then failed.

How to fix this issue?

**Comment by quic-ssiddego** [comment]
Hi @cookcodes The issue is caused because of the usage of currently unsupported "BatchNorm" type in AIMET. Is there a way you could possibly replace the Batchnorms in your model to be of type : [FusedBacthNormV3](https://www.tensorflow.org/api_docs/python/tf/keras/layers/BatchNormalization) and retry.

**Comment by quic-mangal** [comment]
Closing this issue due to inactivity. Please re-open it/ create a new issue if you need further help.
